server/api/student/intervention/intervention.controller.js

```diff
@@ -10,10 +10,13 @@ var Student = require('../student.model');
  * restriction: 'teacher'
  */
 exports.index = function(req, res) {
-  Intervention.find({student: req.student.id}, function(err, interventions) {
-    if (err) return handleError(res, err);
-    return res.status(201).json(interventions);
-  });
+  Intervention
+    .find({student: req.student.id})
+    .sort({_id: -1})
+    .exec(function(err, interventions) {
```

Comment:
Is the date specified in the mongodbId more specific? Or is it sorting on other criteria?